.ddev/web-build/openmage.cron

```diff
@@ -0,0 +1 @@
+* * * * * /var/www/html/cron.sh
```

Comment:
A crontab file must end with EOF. Please insert a newline. Without it you will get the error bellow: 

```php
#20 [web 10/10] RUN { cat /etc/cron.d/*.cron; } | crontab -u ubuntu -
#20 0.349 new crontab file is missing newline before EOF, can't install.
#20 ERROR: process "/bin/bash -c { cat /etc/cron.d/*.cron; } | crontab -u ${username} -" did not complete succes
sfully: exit code: 1
------
 > [web 10/10] RUN { cat /etc/cron.d/*.cron; } | crontab -u ubuntu -:
0.349 new crontab file is missing newline before EOF, can't install.
------
', stderr='failed to solve: process "/bin/bash -c { cat /etc/cron.d/*.cron; } | crontab -u ${username} -" did no
t complete successfully: exit code: 1' 

```